cli/mtv_api_tests/common.py

```
@@ -748,7 +775,9 @@ def mask_passwords(config: dict[str, Any]) -> dict[str, Any]:
     """
     masked: dict[str, Any] = {}
     for key, value in config.items():
-        if isinstance(value, dict):
+        if key == "storage_secret_extra" and isinstance(value, dict):
+            masked[key] = {secret_key: "***" for secret_key in value}
```

Ruff (0.15.14)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)